sawmill-core/src/main/java/io/logz/sawmill/ProcessorFactoriesLoader.java

```diff
@@ -69,7 +59,18 @@ public void loadAnnotatedProcessors(ProcessorFactoryRegistry processorFactoryReg
                 .filter(constructor -> constructor.isAnnotationPresent(Inject.class)).findFirst();
         if (injectConstructor.isPresent()) {
             Class<?>[] servicesToInject = injectConstructor.get().getParameterTypes();
-            Object[] servicesInstance = Stream.of(servicesToInject).map(services::get).toArray();
+            Object[] servicesInstance = Stream.of(servicesToInject)
```

Comment:
This is not clear. What kind of dependency are we looking for here? And why is it all in one PR?

Reply:
It was always there actually

we use this **existing** mechanics to inject geo ip configuration (which points to mmdb path)